stairsplus/common.lua

```diff
@@ -30,16 +30,27 @@ local wall_right_dirmap = {9, 18, 7, 12}
 local wall_left_dirmap = {11, 16, 5, 14}
 local ceil_dirmap = {20, 23, 22, 21}
 
+-- extract the stairsplus category from a node name
+-- assumes the name is in the form mod_name:category_original_ndoe_name
+local function name_to_category(name)
+	local colon, _ = name:find(":") or 0
```

Comment:
Just remove this to fix LuaCheck.
```suggestion
	local colon = name:find(":") or 0
```

Reply:
I'll do but this is pretty stupid honestly, _ is the enstablished lua way to indicate an ignored parameter. Can luacheck be configured to take this into account?

Reply:
You could do this instead:

```diff
- local colon, _ = name:find(":") or 0
+ local colon, _ = name:find(":") or 0 -- luacheck: ignore
```

Reply:
Thanks but I still think luacheck should be fixed, maybe I'll open a PR to fix that there ;-)